pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -5396,3 +5396,87 @@ func (r *ClickHouseReader) GetActiveTimeSeriesForMetricName(ctx context.Context,
 	}
 	return timeSeries, nil
 }
+
+func (r *ClickHouseReader) ListSummaryMetrics(ctx context.Context, req *metrics_explorer.SummaryListMetricsRequest) (*metrics_explorer.SummaryListMetricsResponse, *model.ApiError) {
+	var args []interface{}
+
+	// Convert time range to Unix timestamps
+	startUnix, _ := time.Parse("2006-01-02T15:04:05Z", req.StartDate)
+	endUnix, _ := time.Parse("2006-01-02T15:04:05Z", req.EndDate)
+
+	// Build filters dynamically
+	conditions, _ := utils.BuildFilterConditions(&req.Filters, "t")
+
+	// Build ordering dynamically
+	orderByClause := ""
+	if len(req.OrderBy) > 0 {
+		orderParts := []string{}
+		for _, order := range req.OrderBy {
+			orderParts = append(orderParts, fmt.Sprintf("%s %s", order.ColumnName, order.Order))
+		}
+		orderByClause = "ORDER BY " + strings.Join(orderParts, ", ")
+	}
+
+	whereClause := strings.Join(conditions, " AND ")
+
+	// Final query with simplified structure
+	query := fmt.Sprintf(`
+		SELECT 
+			t.metric_name AS metric_name,
+			argMax(t.description, t.fingerprint) AS description,
+			argMax(t.type, t.fingerprint) AS type,
```

Comment:
What is argMax doing? There is any https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/any aggregate function